scripts/avocado

```diff
@@ -56,10 +56,6 @@ def handle_exception(*exc_info):
     sys.exit(-1)
 
 
-if __name__ == '__main__':
-    sys.excepthook = handle_exception
```

Comment:
This check was on purpose moved here to provide nicer message on `AvocadoApp` import crash, so please don't remove it...